### PR TITLE
Lesson 14 Wiki Notes

### PR DESCRIPTION
**Pull request by gildedgardenia**

### CODE REVIEW

File: Class-14-(SQL-Part-1).md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Class 14: Databases and SQL
```
Comment (jimflores5) [inline Class-14-(SQL-Part-1).md:1]: Don't forget to update the home page to add the class 14 link.

File: Class-14-(SQL-Part-1).md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+## In the prep work this week, the students learned
+1. What a relational database is.
+1. What SQL is and why it is important.
+1. How to perform CRUD operations on a database.
+1. How joins work.
+
+## Announcements
+1. Students are over half way done with Unit 2. Give them a shout out!
+1. Graded Assignment #4 covers the material from Classes 14-17. As always encourage the students to take the time to read through the whole assignment and see what they can get started on now.
```
Comment (jimflores5) [inline Class-14-(SQL-Part-1).md]: Comma after "As always".

File: Class-14-(SQL-Part-1).md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+1. Review what a relational database is. Databases seem simple, but can be kind of weird. Go over how tables store one type of info and are connected to each other.
+1. Review the different CRUD operations and retiterate warnings for updating and deleting data.
+1. Joins can be conceptually odd. Take the time on each one to go over a query and have students discuss what the result set might look like before sharing the result set.
+1. Review the studio!
+1. Before students head out for studio time, let students know that if they had issues installing MySQL and/or MySQL Workbench, resolving those issues come first!
+
+## Small Group Time: Class 14 Studio (TA Notes)
+1. Help students who had issues with installations get going!
+1. Remind your students to follow the sequence of the instructions. If they don't, it is very easy to get tripped up later!
+1. The first sections go over setting up the database in MySQL Workbench. Try to check in with any students that are done with this to make sure they did it correctly! If they didn't and try to start querying, a lot of different errors can arise!
```
Comment (jimflores5) [inline Class-14-(SQL-Part-1).md]: "...with any students who are..."

File: Class-14-(SQL-Part-1).md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+## Large Group Time (Instructor)
+
+### Class 14 Topics That Require Careful Attention
+1. Review what a relational database is. Databases seem simple, but can be kind of weird. Go over how tables store one type of info and are connected to each other.
+1. Review the different CRUD operations and retiterate warnings for updating and deleting data.
+1. Joins can be conceptually odd. Take the time on each one to go over a query and have students discuss what the result set might look like before sharing the result set.
+1. Review the studio!
+1. Before students head out for studio time, let students know that if they had issues installing MySQL and/or MySQL Workbench, resolving those issues come first!
+
+## Small Group Time: Class 14 Studio (TA Notes)
```
Comment (jimflores5) [inline Class-14-(SQL-Part-1).md:23]: I'd add a caution that the students need to use the SECOND SQL command for setting up the ``movies`` table. The first example does not establish a foreign key.

**Review by jimflores5 — APPROVED** [review]
Nice work.  Just two typo fixes and one additional TA tip.

File: Class-14-(SQL-Part-1).md (hunk 1 above)
Comment (gildedgardenia) [reply Class-14-(SQL-Part-1).md:1]: Done! Thanks!

File: Class-14-(SQL-Part-1).md (hunk 2 above)
Comment (gildedgardenia) [reply Class-14-(SQL-Part-1).md]: Done! Thanks!

File: Class-14-(SQL-Part-1).md (hunk 3 above)
Comment (gildedgardenia) [reply Class-14-(SQL-Part-1).md]: Done! Thanks!

File: Class-14-(SQL-Part-1).md (hunk 4 above)
Comment (gildedgardenia) [reply Class-14-(SQL-Part-1).md:23]: Done! Thanks!

**Pull request merged**